src/Common/Wpf/Impl/Collections/RangeObservableCollection.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Collections.Specialized;
+
+namespace Microsoft.Common.Wpf.Collections {
+    /// <summary>
+    /// ObservableCollection that supports bulk changes
+    /// </summary>
+    /// <typeparam name="T"></typeparam>
+    public class RangeObservableCollection<T> : ObservableCollection<T> {
```

Comment:
In light of the renaming below, it feels like this could use a clearer name, too. `BatchObservableCollection`?